src/lib/comms/include/sol-coap.h

```diff
@@ -504,6 +505,25 @@ bool sol_coap_server_is_secure(const struct sol_coap_server *server);
 struct sol_coap_server *sol_coap_server_new(const struct sol_network_link_addr *addr, bool secure);
```

Comment:
Add to the docs that this will create a server using PSK. And the recommend function is sol_coap_server_new_by_cipher_suites from now on.